packages/cli/src/ui/components/ThemeDialog.tsx

```diff
@@ -196,6 +196,7 @@ export function ThemeDialog({
             onSelect={handleThemeSelect}
             onHighlight={onHighlight}
             isFocused={currenFocusedSection === 'theme'}
+            showNumbers={false}
```

Comment:
Since the theme dialog has more than 9 entries, this can be tricky to solve so I've opted to disabled it